CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 - Bug in selecting baselines on a UVData object using `bls` keyword with 3-tuples and
 more than one polarization (introduced in 3.1.2).
+- Bug in the `look_in_catalog` utility function that expected keys which could be
```

Comment:
This should actually go at the top of the stack (entries should be in reverse chronological order).

Reply:
oh, 2 seconds